### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by lmroz**
Summary of changes:
- Updated readme to match new schema.
- Described available configuration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# snap plugin collector - processes
+# Snap plugin collector - processes
 
-snap plugin for collects information about process states grouped by name. Additionally it provides metrics for each running process, also grouped by name. 
+This plugin or [Snap Telemetry Framework](http://github.com/intelsdi-x/snap) collects information about process states grouped by name. Additionally it provides metrics for each running process, also grouped by name.
```
Comment (katarzyna-z) [inline README.md]: It seems that it should be "This plugin for" instead of "The plugin **or**"

File: README.md [hunk 2]
```diff
@@ -16,20 +16,17 @@ snap plugin for collects information about process states grouped by name. Addit
...
-#### Download processes plugin binary:
-You can get the pre-built binaries for your OS and architecture at snap's [Github Releases](https://github.com/intelsdi-x/snap/releases) page.
 
-#### To build the plugin binary:
-Fork https://github.com/intelsdi-x/snap-plugin-collector-processes
+#### Download the plugin binary:
 
+You can get the pre-built binaries for your OS and architecture from the plugin's [GitHub Releases](https://github.com/intelsdi-x/snap-plugin-collector-processes/releasess) page. Download the plugin from the latest release and load it into `snapd` (`/opt/snap/plugins` is the default location for Snap packages).
+#### To build the plugin binary:
+Use https://github.com/intelsdi-x/snap-plugin-collector-processes or your fork as repo.
```
Comment (katarzyna-z) [inline README.md]: Please add a new line after this sentence, it will look better.

File: README.md (hunk 1 above)
Comment (lmroz) [inline README.md]: Done

File: README.md (hunk 2 above)
Comment (lmroz) [inline README.md]: Done

**Comment by katarzyna-z** [comment]
LGTM

**Pull request merged**